src/utils/update/review.beneficial.owner.ts

```diff
@@ -0,0 +1,106 @@
+import { BeneficialOwnerOther, BeneficialOwnerOtherKey } from "../../model/beneficial.owner.other.model";
+import {
+  REVIEW_BENEFICIAL_OWNER_INDEX_PARAM,
+  UPDATE_AN_OVERSEAS_ENTITY_URL, UPDATE_REVIEW_BENEFICIAL_OWNER_INDIVIDUAL_PAGE, UPDATE_REVIEW_BENEFICIAL_OWNER_OTHER_PAGE,
+} from "../../config";
+import { ApplicationData } from "../../model";
+import { BeneficialOwnerIndividual, BeneficialOwnerIndividualKey } from "../../model/beneficial.owner.individual.model";
+
+const AllBoTypes = {
+  boiReview: "review_beneficial_owners_individual",
+  boIndividual: BeneficialOwnerIndividualKey,
+  booReview: "review_beneficial_owners_corporate",
+  boOther: BeneficialOwnerOtherKey
+};
+
+const beneficialOwnerIndividualReviewRedirectUrl = `${UPDATE_AN_OVERSEAS_ENTITY_URL
+      + UPDATE_REVIEW_BENEFICIAL_OWNER_INDIVIDUAL_PAGE
+      + REVIEW_BENEFICIAL_OWNER_INDEX_PARAM}`;
+
+const beneficialOwnerOtherReviewRedirectUrl = `${UPDATE_AN_OVERSEAS_ENTITY_URL
+        + UPDATE_REVIEW_BENEFICIAL_OWNER_OTHER_PAGE
+        + REVIEW_BENEFICIAL_OWNER_INDEX_PARAM}`;
+
+const checkBoIndividualValidation = (boi: BeneficialOwnerIndividual): boolean => {
+  return boi?.usual_residential_address ? true : false;
+};
+
+// this check is to determine whether the BO has been fully submitted
```

Comment:
maybe comment on line 24 instead of 28 as applies to both?

Reply:
to be addressed in Olayode's #869